app/models/concerns/issued_at.rb

```diff
@@ -5,23 +5,18 @@ module IssuedAt
   extend ActiveSupport::Concern
 
   included do
-    before_create :initialize_issued_at
-    before_save :initialize_issued_at
     scope :by_issued_at, ->(issued_at) { where(issued_at: issued_at.beginning_of_month..issued_at.end_of_month) }
     scope :for_year, ->(year) { where("extract(year from issued_at) = ?", year) }
+    validates :issued_at, presence: true
     validate :issued_at_cannot_be_before_2000
     validate :issued_at_cannot_be_further_than_1_year
   end
 
   private
 
-  def initialize_issued_at
-    self.issued_at ||= created_at&.end_of_day
-  end
-
   def issued_at_cannot_be_before_2000
     if issued_at.present? && issued_at < Date.new(2000, 1, 1)
-      errors.add(:issued_at, "Cannot be before 2000")
+      errors.add(:issued_at, "cannot be before 2000")
```

Comment:
Purely cosmetic change.

Error message before: "Issue date Cannot be before 2000"
Error message after:    "Issue date cannot be before 2000"